api/common_stuff.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// listPods returns list of pods in all namespaces.
+func listPods(client *kubernetes.Clientset, fieldSelector string, labelSelector string) ([]v1.Pod, error) {
```

Comment:
Is this func necessary? It looks out of place (hence common), we calls this from 3 places and we don't use any selectors at all at 2 of those. I suggest we just use `client.CoreV1().Pods("").List()` where it is needed atm.

Reply:
This isn't a new function, I just moved it to a common place. I wanted to call it `shit.go`, but that's not PC. You are right, but this function should be destroyed in a subsequent PR.